sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/logging/HttpRequestLoggingInterceptor.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.spring.boot.web.logging;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+
+@Component
+public class HttpRequestLoggingInterceptor extends HandlerInterceptorAdapter {
+
+  private static final Logger logger = LoggerFactory.getLogger(HttpRequestLoggingInterceptor.class);
+
+  @Override
+  public boolean preHandle(
+      HttpServletRequest request, HttpServletResponse response, Object handler) {
+    return true;
+  }
+
+  @Override
+  public void afterCompletion(
```

Comment:
Do we have a follow up subtask for logging some structured information? I just added an example of a Dropwizard Service to the story. Unfortunately we missed to declare most of the fields in the story explicitly. But we added requirements about the trace token.

I assume, we will not be able to provide the same structure but only have the option to put all fields in the SLF4J MDC.